src/test/java/de/qabel/desktop/ui/AbstractControllerTest.java

```diff
@@ -131,11 +134,13 @@ public void setUp() throws Exception {
         diContainer.put("boxSyncIndexFactory", syncIndexFactory);
         diContainer.put("aboutFilesContent", new FilesAbout());
 
+        diContainer.put("boxClient", boxClient);
+
         syncDaemon = new SyncDaemon(new SimpleListProperty<>(), new FakeSyncerFactory());
         diContainer.put("syncDaemon", syncDaemon);
         diContainer.put("accountingUri", new URI("http://localhost:9696"));
 
-        AfterburnerInjector.setConfigurationSource(key -> diContainer.get((String)key));
+        AfterburnerInjector.setConfigurationSource(key -> diContainer.get((String) key));
```

Comment:
You should set your IDE to only reformat vcs modified lines. Your diffs contain a lot of only reformatted code